addons/l10n_ro_edi/models/account_edi_xml_ubl_ciusro.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, _
+
+
+SECTOR_RO_CODES = ('SECTOR1', 'SECTOR2', 'SECTOR3', 'SECTOR4', 'SECTOR5', 'SECTOR6')
+
+
+class AccountEdiXmlUBLRO(models.AbstractModel):
+    _inherit = "account.edi.xml.ubl_bis3"
+    _name = "account.edi.xml.ubl_ro"
+    _description = "CIUS RO"
+
+    def _export_invoice_filename(self, invoice):
+        return f"{invoice.name.replace('/', '_')}_cius_ro.xml"
+
+    def _get_partner_address_vals(self, partner):
+        # EXTENDS account.edi.xml.ubl_bis3
+        vals = super()._get_partner_address_vals(partner)
+
+        if partner.state_id:
+            vals["country_subentity"] = partner.country_code + '-' + partner.state_id.code
```

Comment:
`'country_subentity': partner.state_id.name,`
I really doubt this is necessary to put a custo for that. Did you check regarding a proper validator like ecosio?

Reply:
that's one of the rule from the schematron. it has to be in that format.
![image](https://github.com/odoo/odoo/assets/87525599/d74db77a-07ff-4a24-bba8-b2717eaa0c39)